code/modules/orbit/orbit.dm

```diff
@@ -43,6 +43,10 @@
 	if (!orbiter.orbiting) //admin wants to stop the orbit.
 		orbiter.orbiting = src //set it back to us first
 		orbiter.stop_orbit()
+	var/atom/movable/AM = orbiting
+	if(istype(AM) && AM.orbiting.orbiting == orbiter)

```

Comment:
is the istype necessary?

Reply:
Only movables can orbit but you can orbit turfs too.

Reply:
ew